### DISCUSSION TITLE
Is it possible to change the "dependsOn" retry interval?

### DISCUSSION THREAD

**Discussion by jcheroske** [fluxcd/flux2#4800] [Q&A]
I've set the `interval`, `retryInterval`, and `timeout` values on my kustomization, but none of those seem to affect the 30s `dependsOn` retry interval. Is there a way to change that?

**Comment by stefanprodan** [comment]
Yes there is a `--requeue-dependency` controller flag that controls this, see an example here: https://fluxcd.io/flux/installation/configuration/vertical-scaling/#increase-the-number-of-workers-and-limits
